config/locales/interface/slides/en_supply.yml

```diff
@@ -614,72 +615,8 @@ en:
-        hydrogen, and electricity. This option can be chosen in the
-        <a href=\"/scenario/emissions/ccus/utilisation-and-storage-of-co2\"
-        > CCUS section</a>. "
-    supply_transport_international_shipping:
-      title: International navigation
-      short_description:
-      description: The transport fuels for international navigation depend directly
-        on the type of ship engines you choose. In the <a href="/scenario/demand/transport_international_transport/international-navigation-technology"
-        >international navigation section</a> you can select the engines (HFO or LNG)
-        your international ships will be equipped with. Note that use biofuels is
-        not yet possible for this sector.
-    supply_transport_marine_fuels:
-      title: Domestic navigation
-      short_description: ''
-      description: |
-        Which fuels will be used to power ships with diesel engines and ships with LNG engines?
-        In the <a href="/scenario/demand/transport_freight_transport/domestic-navigation-technology"
-        >domestic navigation</a> section you can determine the share of these engines in the total
-        freight transport by inland shipping.
-    supply_transport_public_transport_technologies:
-      title: Public transport fuels
-      short_description:
-      description:
-    supply_transport_road_transport:
-      title: Road transport
-      short_description:
-      description: |
-        How much of the fuels used in road transport will come from renewable
-        sources? Do consider the impact on land (and sea) use for growing the required
-        crops. By changing the road transport technologies in the
-        <a href="/scenario/demand/transport_passenger_transport/car-technology">passenger</a>
-        and <a href="/scenario/demand/transport_freight_transport/truck-technology">freight</a>
-        transport section, you can determine which engines, and therefore which transport fuels
-        are used.
-    supply_transport_trains:
-      title: Rail transport
-      short_description: ''
-      description: How much of the fuel used by diesel trains will come from renewable
-        sources? Do not forget that the arable land used for growing the required
-        crops for biodiesel competes with food production. In the <a href="/scenario/demand/transport_passenger_transport/train-technology"
-        >passenger train section</a> and <a href="/scenario/demand/transport_freight_transport/train-technology"
-        >freight train section</a> you can specify your train technology.
-    future_fuel_production:
-      title: Future production of primary carriers
+    supply_fuel_extraction:
+      title: Extraction of primary fuels
       short_description: ''
       description: |
         Coal, oil, gas and uranium oxide are defined as ‘primary carriers’ of energy and are converted to other energy carriers such as electricity, heat or gasoline.
```

Comment:
Perhaps we should also update the slide text here? It is quite long and contains a lot of information that relates to a chart and should therefore be in the chart description.